src/components/ebay-spark-line/test/test.server.js

```diff
@@ -0,0 +1,43 @@
+import { expect, use } from 'chai';
+import { render } from '@marko/testing-library';
+import template from '..';
+
+use(require('chai-dom'));
+
+const mock = {
+    data: [
+        {
+            x: 0,
+            y: 0,
+        },
+        {
+            x: 1,
+            y: 1,
+        },
+    ],
+};
+describe('spark-line', () => {
+    it('renders a blue line', async () => {
+        const { getByRole } = await render(template, mock);
+        const line = getByRole('graphics-symbol');
+        expect(line.classList.contains('ebay-spark-line__path-neutral'));
+    });
```

Comment:
I'm happy to see tests.
Please look up `button` server tests. We have snapshot testing we use which is better than checking for class existence. 
We also use storybook as the basis of our mocks (see `button` again)

Some components are not changed, but we are going in that direction.